src/components/navigation/navigation/navigation.ts

```diff
@@ -169,6 +172,14 @@ export class SbbNavigationElement extends UpdateScheduler(LitElement) {
     activeAction?.connectedSection?.open();
   }
 
+  private _checkActiveActions(): void {
+    const activeActions = Array.from(this.querySelectorAll('sbb-navigation-button.sbb-active')) as (
```

Comment:
otherwise create a @hostAttributes on mixin like with buttons and create a data-navigation-
```suggestion
    const activeActions = Array.from(this.querySelectorAll(':is(sbb-navigation-button, sbb-navigation-link).sbb-active')) as (
```